### PR TITLE
Firebase test rt

### PR DESCRIPTION
**Pull request by r7uaz0n**
Here is my commit and pull request for today's Firebase app challenge.

### CODE REVIEW

**Comment by kammitama5** [comment]
Looking good!

**Comment by LearningNerd** [comment]
Nice! Good work, @r7uaz0n! I'll post more steps soon so hold onto this branch for now. :)

File: firebase-test/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing Firebase!</title>
+  </head>
+  
+  <body>
+  	<h1>Testing Firebase!</h1>
+  	 <p id="message"></p>
+  	 
+  	<script src="https://www.gstatic.com/firebasejs/3.9.0/firebase.js"></script>
+  	<script src="firebasetest.js"></script>.
```
Comment (LearningNerd) [inline firebase-test/index.html]: I see a stray period at the end of this line =P probably should delete that

Comment (r7uaz0n) [inline firebase-test/index.html]: Oops,  I see that. I will fix.

**Pull request closed without merging**